doc/lua_api.md

```diff
@@ -8257,14 +8257,13 @@ child will follow movement and rotation of that bone.
                 `"default"` uses the classic Minetest sun and moon tinting.
                 Will use tonemaps, if set to `"default"`. (default: `"default"`)
         * `fog`: A table with following optional fields:
-            * `fog_distance`: integer, set an upper bound the client's viewing_range (inluding range_all).
-               By default, fog_distance is controlled by the client's viewing_range, and this field is not set.
-               Any value >= 0 sets the desired upper bound for the client's viewing_range and disables range_all.
-               Any value < 0, resets the behavior to being client-controlled.
+            * `fog_distance`: integer, set an upper bound the client's viewing_range.
+               Any value >= 0 sets the desired upper bound for the client's viewing_range,
+               disables range_all and disallows disabling fog (F3 key).
```

Comment:
```suggestion
               disables range_all and ignores the setting `keymap_toggle_fog`.
```

Because the key can be changed. Alternatively

```suggestion
               disables range_all and disallows disabling fog by key bindings.
```

Reply:
how about this? I wanted to mention the key so that the reader knows which function this is referring to.
```suggestion
               disables range_all and disallows disabling fog (default: F3 key).
```

Reply:
That has potential to go out-of-sync in case of a default change but I don't think that is likely to happen. Thus, fine by me.
:+1: